### PR TITLE
Update apps repo readme.

### PR DESCRIPTION
**Pull request by HumairAK**
Resolve: https://github.com/operate-first/apps/issues/1532

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,73 @@
...
+
+![](./docs/img/apps_repo.png)
+
+This diagram provides a simple illustration of the structure of a typical `Application` in this repo. In this
+illustration you can imagine an `Application` to be any group of manifests. These manifests belonging to an
+`Application` often make up a full deployment of a service which has end users (e.g. `Grafana`). Though sometimes,
+an `Application` may be just be a way to logically group a collection of manifests, like in the case of `cluster-scope`
+`Application` which harbors all cluster wide resources, configurations, and privileged resources.
+
+Once a group of manifests are organized into a directory, they can be deployed using an ArgoCD. To do this you simply
```
Comment (4n4nd) [inline README.md:33]: remove `an` before `ArgoCD`

File: README.md [hunk 2]
```diff
@@ -1,5 +1,73 @@
...
+
+You will also notice each app adheres to a structure made up of `bases` and `overlays`. This is a structure commonly
+found when using `Kustomize`. `base` directories contain manifests/configurations that are common to multiple clusters
+and generally don't harbor any cluster specific details. Within the `overlays` directory you'll find cluster specific
+configurations. A cluster specific folder within `Overlays` will inherit the manifests from `base` folders as needed,
+while also making patches/amendments specific to that cluster.
+
+## Contributing
+
+We suggest ramping up on the following before attempting to work with the Apps repo:
```
Comment (4n4nd) [inline README.md:46]: how about we make `apps repo` a hyperlink in case this doc is published on the opf website?

**Review by 4n4nd — CHANGES_REQUESTED** [review]
looks awesome!
very minor changes requested

File: README.md (hunk 2 above)
Comment (HumairAK) [reply README.md:46]: good call!

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**